packages/checkout/sdk/src/smartCheckout/fees/fees.ts

```diff
@@ -17,6 +18,8 @@ const calculateFeesPercent = (
 
   const bnFeeAmount = amountBn
     .mul(BigNumber.from(feePercentageMultiplier))
+    .div(BigNumber.from(tokenQuantity))
+    .mul(BigNumber.from(tokenQuantity))
```

Comment:
**Question:** Is the effective result always rounded up or down or to the closest divisible value? Regardless of which one, could we leave a comment here that explains this?